setup.py

```diff
@@ -18,7 +18,8 @@
     url='https://github.com/cea-cosmic/ModOpt',
     download_url='https://github.com/cea-cosmic/ModOpt',
     packages=find_packages(),
-    install_requires=['numpy>=1.16.4', 'scipy>=1.3.0,<=1.3.3', 'progressbar2>=3.34.3'],
+    install_requires=['numpy>=1.16.4', 'scipy>=1.3.0,<=1.3.3',
+                      'progressbar2>=3.34.3'],
```

Comment:
The best way to handle these and not have to rewrite them in the future would be:
```
install_requires=[
  'numpy>=1.16.4',
  'scipy>=1.3.0,<=1.3.3',
  'progressbar2>=3.34.3',
]
```

Reply:
Fair point, but I have instead put everything into a requirements.txt file. This way we won't even have to touch the setup.py file regarding dependencies.

Reply:
sure makes even more sense !